### PR TITLE
Select autopilot during wallet setup.

### LINKED ISSUE #1167: Let users choose if autopilot is enabled during setup
paste

### PR DESCRIPTION
**Pull request by valentinewallace**
Closes #1167. Tested in an iOS build. I didn't add the feature to desktop.

### CODE REVIEW

File: src/view/select-atpl-mobile.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        Automatically set up channels?
+      </CopyOnboardText>
+      <Text style={styles.copyTxt}>
+        {
+          "If you want to avoid manual channel creation, the app can allocate funds to channels for you. It's called autopilot."
+        }
+      </Text>
+      <View style={styles.list}>
+        <SettingItem
+          name="Use autopilot (recommended)"
```
Comment (valentinewallace) [inline src/view/select-atpl-mobile.js]: I added the word "recommended"... Feel free to remove if you prefer.

File: src/component/setting.js [hunk 2]
```diff
@@ -100,9 +109,22 @@ const iStyles = StyleSheet.create({
...
+  children,
+}) => (
+  <ListItem
+    style={[iStyles.item, copy ? { height: 80 } : null]}
+    onSelect={onSelect}
+  >
+    <View style={iStyles.left}>
+      <Text style={iStyles.name}>{name}</Text>
+      {copy ? <Text style={iStyles.copy}>{copy}</Text> : null}
+    </View>
```
Comment (tanx) [inline src/component/setting.js]: Would probably be better to create a separate `SettingCopyItem` component than trying to squeeze both use cases into one component. That would also allow us to set the margins/padding according to the design, as they look pretty different right now.

Design:


Implementation:

File: src/view/select-atpl-mobile.js (hunk 1 above)
Comment (tanx) [reply src/view/select-atpl-mobile.js]: makes sense 👍

**Comment by valentinewallace** [comment]
Updated @tanx :)

File: src/component/setting.js (hunk 2 above)
Comment (valentinewallace) [reply src/component/setting.js]: Thanks, must've been in a hurry hah

**Pull request merged**